Looks like SourceForge timed out:
```
-----------------------------------------------------
Installer Availability
-----------------------------------------------------
Status: Started

6/1/2020 8:27:38 AM
Status: Completed
Result: Failed
Error occurred during installer copy from https://sourceforge.net/projects/clamwin/files/clamwin/0.99.4/clamwin-0.99.4-setup.exe/download.
 A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond

6/1/2020 8:27:45 AM

****************************************************************************************
Installer Validation
****************************************************************************************
Status: Completed
##[error] Result: Failed
```

Can someone retrigger the bot?

--- a/manifests/ClamWin/ClamWin/0.99.4.yaml
+++ b/manifests/ClamWin/ClamWin/0.99.4.yaml
@@ -1,0 +1,13 @@
+Id: ClamWin.ClamWin
+Version: 0.99.4
+Name: ClamWin Free Antivirus
+Publisher: ClamWin Pty Ltd.
+License: GPL
+Tags: virus, anti-virus, security, clamav
+Description: ClamWin is a Free Antivirus program for Microsoft Windows 10 / 8 / 7 / Vista / XP / Me / 2000 / 98 and Windows Server 2012, 2008 and 2003.
+Homepage: http://www.clamwin.com
+Installers:
+  - Arch: x86
+    Url: https://sourceforge.net/projects/clamwin/files/clamwin/0.99.4/clamwin-0.99.4-setup.exe/download
+    Sha256: C6F272AFDB48DB7D5600A1934D2C3B028C4DB76FE96D3D8681383F6A69A33722
+    InstallerType: inno

